### PR TITLE
Clean Github pages repository

### PR DESCRIPTION
**Pull request by ymzayek**
@NicolasGensollen I went ahead and cleaned this repository. Let me know if I removed something I shouldn't have. Everything inside `dev/` and `stable` directories should be untouched.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,14 +0,0 @@
-<!DOCTYPE HTML>
-<html lang="en-US">
-    <head>
-        <meta charset="UTF-8">
-        <meta http-equiv="refresh" content="1; url=https://nilearn.github.io/stable/index.html">
-        <script type="text/javascript">
-            window.location.href = "https://nilearn.github.io/stable/index.html"
```
Comment (NicolasGensollen) [inline index.html:7]: This file is required as it implements the redirection to `stable/index.html`.

**Review by NicolasGensollen — CHANGES_REQUESTED** [review]
Thanks for taking care of this @ymzayek ! 👍 
(and sorry I didn't do it although I said I would... 😓).

File: index.html (hunk 1 above)
Comment (ymzayek) [reply index.html]: Ah ok I see, I added it back.

Comment (ymzayek) [reply index.html]: And what is `index_save.html`? Is that also relevant?

Comment (NicolasGensollen) [reply index.html]: It's a copy of the automatically generated `index.html` (now in `dev` and `stable` folders). I'm not sure why it was kept, but we don't need it anymore.

**Review by NicolasGensollen — APPROVED** [review]
LGTM, thanks @ymzayek !

**Comment by ymzayek** [comment]
Ok to merge here with one approval?

**Comment by bthirion** [comment]
I think you should. It's super hard for me to make sense of the diff.
The main point is to carefully look at the repository pages afterwards.

**Comment by NicolasGensollen** [comment]
Actually, the docs were broken because this PR removed the `.nojekyll` file at the root (which I completely missed when reviewing... 😓).
I added it back the quick and dirty way, docs should be accessible again.
Let me know if you have issues.

**Comment by bthirion** [comment]
Thx!

**Pull request merged**